api/v1/views/amenities.py

```diff
@@ -1 +1,70 @@
 #!/usr/bin/python3
+""" amenities view """
+
+from flask import jsonify, abort, request
+from api.v1.views import app_views
+from models import storage
+from models.amenity import Amenity
+
+
```

Comment:
You seem to be missing the small `all_amenities` function that just retrieves a list of all the current `Amenity` objects